app/controllers/static_controller.rb

```diff
@@ -1,5 +1,11 @@
 class StaticController < ApplicationController
   def index
+    @press = {
+      'fast-company' => 'http://fastcompany.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.